bedrock/pocket/templates/pocket/safari.html

```diff
@@ -8,10 +8,10 @@
 
 {% set platform_image = 'img/pocket/welcome-safari.jpg' %}
 
-{% block page_title %}Pocket for Safari{% endblock %}
+{% block page_title %}{{ ftl('pocket-platform-safari-title') }}{% endblock %}
 
-{% block page_desc %}Installing the Pocket browser extension installs buttons that let you save items with one click.{% endblock %}
+{% block page_desc %}{{ ftl('pocket-platform-browser-installing') }}{% endblock %}
 
-{% block page_cta %}Download{% endblock %}
```

Comment:
**Note:** Safari was the only browser page to have the CTA to say `download` instead of `install`, so I went ahead and changed it to install to match the other browser pages for better consistency.